doc/tutorials/getting_started/getting_started.rst

```diff
@@ -7,10 +7,17 @@ Building all the source code of MoveIt can take 20-30 minutes, depending on the
 
 Install ROS 2 and Colcon
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-:ros_documentation:`Install ROS 2 {DISTRO_TITLE}<Installation.html>`.
-It is easy to miss steps when going through the ROS 2 installation tutorial. If you run into errors in the next few steps, a good place to start is to go back and make sure you have installed ROS 2 correctly.  One that users commonly forget is to source the ROS 2 install itself.  ::
+MoveIt 2 currently supports multiple versions of ROS.
+Install whichever version you prefer.
```

Comment:
Should we recommend that users [build docker images](https://moveit.picknik.ai/main/doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.html) with `main` on the ROS distro of their choice?

Reply:
That is already recommended above this as an alternative.